docs/contributors/release.md

```diff
@@ -192,22 +192,23 @@ WordPress Core Updates are based on the `g-minor` branch. Releasing packages in
 
 For major WordPress releases, the last Gutenberg plugin release is merged into `g-minor`. This involves the following steps:
 
-1. Checkout the last published Gutenberg's release branch `git checkout release/x.x`
+1. Check out the last published Gutenberg release branch `git checkout release/x.x`
 2. Create a Pull Request from this branch into `g-minor`.
 3. Merge the branch.
 
 ### Minor WordPress Releases
 
-For minor releases, the critical fixes targetted for this WordPress Minor release should be cherry-picked into the `g-minor` branch one by one in their chronological order.
+For minor releases, the critical fixes targeted for this WordPress Minor release should be cherry-picked onto the `g-minor` branch one by one in their chronological order.
```

Comment:
I was not sure if onto is the right fit here.
Searching for "cherry-picked into the branch" returns 499 results and searching for "cherry-picked onto the branch" returns 8 results, so it seems into the branch is more common.
For learning purposes do you have some rule you normally follow for this case? 
Thank you in advance!

Reply:
Thanks for checking. I'm fine with reverting this. :)

> For learning purposes do you have some rule you normally follow for this case?

I followed the case for a physical tree branch: I would sit _on_ a branch and not _in_ one. But that doesn't mean it would apply to Git jargon. Perhaps a native speaker could provide some perspective.

Reply:
Neither really draws my attention as being more right than the other. To me, "into" might feel more natural only in that cherry-picking feels like the act of pulling _in_ some commit made elsewhere, regardless of whether that commit is being placed _on_ the existing branch history.

Reply:
Thank you both. :) I will `s/o/i/`.